Widgets/ArenaWidget.lua

```diff
@@ -17,7 +17,13 @@ local pairs = pairs
 local UnitExists = UnitExists
 local IsInInstance = IsInInstance
 local IsInBrawl = C_PvP.IsInBrawl
+local UnitIsUnit = UnitIsUnit
+local UnitInParty = UnitInParty
+local UnitName = UnitName
 local MAX_ARENA_ENEMIES = MAX_ARENA_ENEMIES or 5 -- MAX_ARENA_ENEMIES is not defined in Wrath Clasic
+local GetAddOnEnableState = (C_AddOns and C_AddOns.GetAddonEnableState)
```

Comment:
There is a type here. Correct is C_AddOns.GetAddOnEnableState (Addon => AddOn)

Reply:
Fixed.